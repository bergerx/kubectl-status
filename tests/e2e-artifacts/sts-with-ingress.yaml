apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: sts-with-ingress
spec:
  replicas: 1
  selector:
    matchLabels:
      app: sts-with-ingress
  template:
    metadata:
      labels:
        app: sts-with-ingress
    spec:
      containers:
      - name: sts-with-ingress
        image: registry.k8s.io/pause:3.9
---
apiVersion: v1
kind: Service
metadata:
  name: sts-with-ingress
spec:
  selector:
    app: sts-with-ingress
  ports:
  - protocol: TCP
    port: 80
    targetPort: 80
---
apiVersion: networking.k8s.io/v1
kind: Ingress
metadata:
  name: sts-with-ingress
spec:
  rules:
  - host: sts-with-ingress.com
    http:
      paths:
      - path: /
        pathType: Prefix
        backend:
          service:
            name: sts-with-ingress
            port:
              number: 80
