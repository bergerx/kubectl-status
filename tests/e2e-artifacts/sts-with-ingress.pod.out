
Pod/sts-with-ingress-0 -n default, created 1m ago by StatefulSet/sts-with-ingress Running BestEffort
  Current: Pod is Ready
  Managed by sts-with-ingress application
  PodScheduled -> Initialized -> ContainersReady -> Ready for 1m
  Containers:
    sts-with-ingress (registry.k8s.io/pause:3.9) Running for 1m and Ready
  Known/recorded manage events:
    1m ago Updated by kube-controller-manager (metadata, spec)
    1m ago Updated by kubelet (status)
  Services matching this pod:
    Service/sts-with-ingress -n default, created 1m ago, last endpoint change was 1m ago
      Current: Service is ready
      Ready: Pod/sts-with-ingress-0 -n default on Node/minikube, 1.1.1.1:80/TCP
      Known/recorded manage events:
        1m ago Updated by kubectl-client-side-apply (metadata, spec)
      Ingresses matching this Service:
        Ingress/sts-with-ingress -n default, created 1m ago, gen:1
          Current: Resource is current
          Service/sts-with-ingress:80 has 1 endpoints.
          Known/recorded manage events:
            1m ago Updated by kubectl-client-side-apply (metadata, spec)
